src/util/ParseableDuration.h

```diff
@@ -17,8 +17,12 @@ namespace ad_utility {
 
 // Wrapper type for std::chrono::duration<> to avoid having to declare
 // this in the std::chrono namespace.
-template <typename DurationType>
+template <typename DT>
 class ParseableDuration {
```

Comment:
You can just continue calling the template parameter `DurationType`, right?

Reply:
Not if I want to have public using directive with the same name